### PR TITLE
Milestone 1: Work and About section

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
## Portfolio milestone 1: Work and About section

In this section I : 
- Add work section to the portfolio homepage
- Add about section to the portfolio homepage
- Implement GitHub flow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,6 +50,182 @@ <h2>LET’S CONNECT</h2>
...
+                  <li class="tag1"><h3>HTML</h3></li>
+                  <li class="tag2"><h3>CSS</h3></li>
+                  <li class="tag3"><h3>Javascript</h3></li>
+                </ul>
+              </div>
+              <div class="see-more-detail">
+                <div class="button">
+                  <a href="#">
+                    See Project
+                  </a>
```
Comment (TedLivist) [inline index.html:77]: Please correct your design issue as described below. Do this for other parts of your code as well that has this issue.
- [ ] Your `javascript` tag falls out its border
- [ ] Please add more `padding-right` to your **`See Project`** links to look more like the template

emma-js-btns

File: index.html [hunk 2]
```diff
@@ -49,6 +50,182 @@ <h2>LET’S CONNECT</h2>
...
+            <h3>Multi-Post
+              Stories</h3>
+            <div class="frame2">
+              <ul>
+                <li class="client"><h4>CANOPY</h4></li>
+                <li class="role"><i class="fa fa-circle"></i><h4>Back End Dev</h4></li>
+                <li class="year"><i class="fa fa-circle"></i><h4>2015</h4></li>
+              </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```
Comment (TedLivist) [inline index.html:127]: - [ ] Your font for the last two paragraphs of the projects look different from others and the template. Please, use the correct font.

emma-font

File: assets/css/styles.css [hunk 3]
```diff
@@ -373,21 +255,1153 @@ header {
...
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
+
+.resume:active {
+  top: calc(50% - 48px / 2 + 57.5px);
+  background: #2230d2;
+  border-radius: 8px;
+}
```
Comment (TedLivist) [inline assets/css/styles.css:1136]: -  [ ] Please change the color of the 'Get My Resume' text when hovered or active as defined in the template.

**Currently, when active it looks like this:**
emma-get-resume

**But the template has this:**

btns-styles-2

File: index.html [hunk 4]
```diff
@@ -49,6 +50,182 @@ <h2>LET’S CONNECT</h2>
...
+                <li><a href="#"><img src="assets/images/mobile/m.svg" class="about-m-icon" alt="image-about"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/finger.svg" class="finger-icon-about" alt="image"></a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="resume">
+            <a href="#">
+              Get my resume
+            </a>
+          </div>
```
Comment (TedLivist) [inline index.html:202]: -  [ ] Add more right padding to your 'Get my resume' link to look similar to what is on the template.

File: index.html [hunk 5]
```diff
@@ -49,6 +50,182 @@ <h2>LET’S CONNECT</h2>
...
+              <h3>Frameworks</h3>
+              <img src="assets/images/mobile/Icon-Right-Larger.svg" alt="">
+            </li>
+            <li class="skill">
+              <h3>Skills</h3>
+              <img src="assets/images/mobile/Icon-Right-Larger.svg" alt="">
+            </li>
+          </ul>
+        </div>
+      </div>
```
Comment (TedLivist) [inline index.html:227]: Please improve the design of the section in the image
- [ ] **Languages**, **Frameworks** and **Skills** should be bolder as defined in the template
- [ ] **Frameworks** and **Skills** should some kind of border-bottom as in the template
- [ ] The text **HTML** should be somewhat vertically aligned in the middle just like **Javascript** and **CSS**
- [ ] The images should have some top and left margins as in the template.

emma-langs

**Review by TedLivist — CHANGES_REQUESTED** [review]
Hi @Emmy-github-webdev,

Good job so far!
Your design looks good ✔️ 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -28,7 +29,7 @@
     <section class="header-page">  
       <div class="header-container">
           <div class="headline">
-            <h1>I’m Emmanuel Glad to see you!</h1>
+            <h1>I’m Emmanuel Glad to see <br/> you!</h1>
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
             <div class="social-media">
```
Comment (Mihndim2020) [inline index.html:35]: - [ ] As requested by the former code reviewer, which is a requirement for this project, and a best practice to stick to the client's design, please kindly position your social media icons as in the design.  The margin between "LET'S CONNECT" text and the social media icons is too large. Please kindly fix that.

***The design***
emmyd

***Your implementation***
emmyI

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @Emmy-github-webdev, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — APPROVED** [review]
### Hi @Emmy-github-webdev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**